docker/consumedir.dockerfile

```diff
@@ -1,19 +1,13 @@
 ## CONSUMEDIR
 
-ARG VERSION=
-ARG REPO=
-
-# hack to use args in from
-FROM ${REPO}:base-binaries-${VERSION} as docspell-base-binaries
-
-
-FROM ${REPO}:base-${VERSION}
+FROM alpine:latest
+LABEL maintainer="eikek0 <eike@docspell.org>"
 
 RUN apk add --no-cache curl bash inotify-tools
 
-COPY --from=docspell-base-binaries /opt/docspell-tools /opt/docspell-tools
+COPY ./tools/consumedir /opt/consumedir
 
-ENTRYPOINT /opt/docspell-tools/consumedir/consumedir.sh --path /opt/docs -i --iheader Docspell-Integration:$DOCSPELL_HEADER_VALUE -m http://docspell-restserver:7880/api/v1/open/integration/item -v
+ENTRYPOINT ["bash", "/opt/consumedir/consumedir-entrypoint.sh"]
```

Comment:
This is great, I think it fixes #507 ? The image can now be used with other params, supplied via env variables, right?

Reply:
Yes, I guess it does. I didn't notice there was an issue for that. Perfect ;-)
I hope I have added env variables for everything though, maybe you can check that again?

Reply:
It's ok if there aren't variables for everything. Can be later added.